app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -90,13 +90,11 @@
       </v-col>
     </v-row>
     <v-alert
-      :color="saving ? 'primary' : undefined"
-      dense
-      text
-      transition="scale-transition"
-      :type="saving ? 'info' : 'success'"
       :value="saved || saving"
-      class="mt-md-0 mt-3"
+      :class="saving ? NOTIFICATIONS_TYPES.INFO.class : NOTIFICATIONS_TYPES.SUCCESS.class"
+      :color="saving ? NOTIFICATIONS_TYPES.INFO.color : NOTIFICATIONS_TYPES.SUCCESS.color"
+      :icon="saving ? NOTIFICATIONS_TYPES.INFO.icon : NOTIFICATIONS_TYPES.SUCCESS.icon"
```

Comment:
There may be situations where saving fails due to network traffic issues or other random acts of chaos. This only handles the successful case and implies success even when something could have failed.

Reply:
But don't address that as part of this enhancement, this should be about specific style changes. Let's make a different ticket if there's an unhandled error case in the display coding.

I'm not sure we need the `color` `class` and `icon` here as well. Vuetify alerts take in a `type` that should define this and then our style overrides would control what that looks like.

Reply:
@jujaga I agree, I want to address that network fail case but we should move this fix in a different ticket since this PR should be for the global Notification/Alert Changes.

Reply:
@loneil After doing some playing around with styling, if you introduce a type property to the v-alert Component it ends up producing too many built in Vuetify styles (such as color, icon, positioning, etc.) that interfere with the BCGov styling that we are trying to achieve.